### PR TITLE
Refactoring and updates

### PR DESCRIPTION
**Pull request by roxblnfk**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

Properties now with types (hello php 7.4); console commands returns int (symfony/console 5 compatibility)

### CODE REVIEW

File: src/Command/UpCommand.php [hunk 1]
```diff
@@ -40,7 +40,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 '<fg=red>Error!</>',
                 $e->getMessage(),
             ]);
-            return;
+            $code = $e->getCode();
+            return is_numeric($code) ? (int)$code : 0;
```
Comment (rustamwin) [inline src/Command/UpCommand.php]:
```suggestion
            return (int)$code;
```

Comment (xepozz) [reply src/Command/UpCommand.php]: Or simply `return $e->getCode();`

File: src/Command/DownCommand.php [hunk 2]
```diff
@@ -36,7 +36,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 '<fg=red>Error!</>',
                 $e->getMessage(),
             ]);
-            return;
+            $code = $e->getCode();
+            return is_numeric($code) ? (int)$code : 0;
```
Comment (xepozz) [inline src/Command/DownCommand.php]: The same https://github.com/yiisoft/yii-cycle/pull/10/files#r354438086

File: src/Command/CreateCommand.php [hunk 3]
```diff
@@ -16,10 +16,12 @@ public function configure(): void
...
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output): void
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $customName = $input->getArgument('name');
 
         $this->createEmptyMigration($output, $customName);
+
+        return 0;
```
Comment (xepozz) [inline src/Command/CreateCommand.php]: I think it should return [ExitCode::OK](https://github.com/yiisoft/yii-console/blob/master/src/ExitCode.php#L32)

File: src/Command/UpCommand.php (hunk 1 above)
Comment (roxblnfk) [reply src/Command/UpCommand.php]: Or `return $e->getCode() ?: ExitCode::UNSPECIFIED_ERROR;`

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**